### ISSUE TITLE
PIP install error

### ISSUE THREAD

**Issue by rayworrell** [Narcolapser/python-o365#77]
Please forgive me...  I'm new when it comes to all of this.  

I was trying to install the o365 library by...  

pip install o365

...and, I got the following errors...

      File "c:\users\rworr\appdata\local\programs\python\python36-32\lib\configparser.py", line 1149, in _unify_values
        return _ChainMap(vardict, sectiondict, self._defaults)
      File "c:\users\rworr\appdata\local\programs\python\python36-32\lib\collections\__init__.py", line 872, in __init__
        self.maps = list(maps) or [{}]          # always at least one map
    RecursionError: maximum recursion depth exceeded while calling a Python object

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in C:\Users\rworr\AppData\Local\Temp\pip-build-iiggw9ts\o365\

Did I do something wrong?

Thanks!

**Comment by Narcolapser** [comment]
No, that looks right. I'll give this a try next time I'm on a windows machine.

**Comment by rayworrell** [comment]
Thank you, sir!

**Comment by Narcolapser** [comment]
Sorry for the delay. I'm trying to look at this now. So far I've not had any success recreating your problem. I have a basically fresh windows install witha  fresh python install and it worked fine. Can you try un-installing and re-installing python to see if that clears it up? I'll keep poking at it on this end to see if I can recreate your problem.

**Comment by Narcolapser** [comment]
if that doesn't work, one other thing you could try is cloning the repository and installing manually:

    git clone https://github.com/Narcolapser/python-o365.git
    cd python-o365
    python setup.py install

**Comment by anish1106** [comment]
Hi,
This could be a stupid suggestion, but I think it should be O365 not o365.

**Comment by Narcolapser** [comment]
at this point it wouldn't be prudent to change it. The library is already well established so changing it or creating a duplicate would only cause confusion. 

Also. please open a new issue next time rather than cluttering an existing one.

**Comment by rayworrell** [comment]
Thanks, Narcolapser for taking a peek at this.  

I tried uninstalling and reinstalling Python and attempting to install the repository manually, but I keep getting the same "maximum recursion depth exceeded" error.  Interestingly, I've gotten that for one other library I attempted to install, so it's definitely an environmental problem of some sort.

It makes me very sad, but thank you for your help.
